src/Native/System.Native/pal_networkstatistics.cpp

```diff
@@ -412,4 +413,113 @@ extern "C" int32_t GetActiveUdpListeners(IPEndPointInfo* infos, int32_t* infoCou
     return 0;
 }
 
+extern "C" int32_t GetNativeIPInterfaceStatistics(char* interfaceName, NativeIPInterfaceStatistics* retStats)
```

Comment:
This is called from internal library code where the interface name should always be a valid name (retrieved separately from getifaddrs, and the retStats pointer should be a pointer to a stack object declared just before the call. I don't think it would hurt to add those assertions, though.